industry_benchmarks/utils/plan_rbfe_network.py

```diff
@@ -0,0 +1,243 @@
+import string
+import click
+import pathlib
+import logging
+import warnings
+import os
+from functools import partial
+from openff.units import unit
+import openfe
+from openfe.protocols.openmm_rfe.equil_rfe_methods import RelativeHybridTopologyProtocol
+from rdkit import Chem
+import kartograf
+from openff.toolkit import (
+    RDKitToolkitWrapper, AmberToolsToolkitWrapper
+)
+from openff.toolkit.utils.toolkit_registry import (
+    toolkit_registry_manager, ToolkitRegistry
+)
+
+logger = logging.getLogger(__name__)
+
+amber_rdkit = ToolkitRegistry([RDKitToolkitWrapper(), AmberToolsToolkitWrapper()])
+
+
+def gen_charges(smc):
+    """
+        Generate AM1BCC partial charges for a SmallMoleculeComponent using
+        the input conformer and antechamber as backend.
+    """
+    offmol = smc.to_openff()
+    with toolkit_registry_manager(amber_rdkit):
+        offmol.assign_partial_charges(partial_charge_method="am1bcc")
```

Comment:
here we should be enforcing the input conformer when assigning partial charges

Reply:
Changed this!